crates/wit-parser/src/ast.rs

```diff
@@ -987,6 +997,16 @@ impl<'a> InterfaceItem<'a> {
                 NamedFunc::parse(tokens, docs, attributes).map(InterfaceItem::Func)
             }
             Some((_span, Token::Use)) => Use::parse(tokens, attributes).map(InterfaceItem::Use),
+            Some((_span, Token::Nest)) => {
+                tokens.eat(Token::Nest)?;
+                let path = UsePath::parse(tokens)?;
+                tokens.expect_semicolon()?;
+                Ok(InterfaceItem::Nest(Nest {
+                    docs,
+                    attributes,
+                    from: path,
+                }))
```

Comment:
Mind moving this into a `Nest::parse` method? (basically preserving the style/structure of this method where it delegates to other top-level parsers)